test/unit/data/model/migration_fixes/conftest.py

```diff
@@ -0,0 +1,47 @@
+from typing import (
+    Generator,
+    TYPE_CHECKING,
+)
+
+import pytest
+from sqlalchemy import (
+    create_engine,
+    text,
+)
+from sqlalchemy.orm import Session
+
+from galaxy import model as m
+
+if TYPE_CHECKING:
+    from sqlalchemy.engine import Engine
+
+from galaxy.model.unittest_utils.model_testing_utils import (  # noqa: F401 - url_factory is a fixture we have to import explicitly
+    sqlite_url_factory,
+)
+
+
+@pytest.fixture()
+def db_url(sqlite_url_factory):  # noqa: F811
+    return sqlite_url_factory()
+
+
+@pytest.fixture()
+def engine(db_url: str) -> "Engine":
+    return create_engine(db_url)
+
+
+@pytest.fixture
+def session(engine: "Engine") -> Session:
+    return Session(engine)
+
+
+@pytest.fixture(autouse=True)
+def clear_database(engine: "Engine") -> "Generator":
+    """Delete all rows from all tables. Called after each test."""
+    yield
+    with engine.begin() as conn:
```

Comment:
Can we put in a safety mechanism here and only run on sqlite ? I worry about accidentally including the autouse fixture and deleting important-ish databases.

Reply:
That's a good point. I'll try to remove the `autouse` option altogether.  Tests may be more verbose, but it's a reasonable trade-off here, I think. (Just doing it for sqlite will prevent us from running any of these tests on postgres; and we don't want to accidentally wipe out sqlite data either.

There's another one, in `test/unit/data/model/db/conftest.py` - I'll do the same.